pkg/extensions/search/common/common_test.go

```diff
@@ -57,6 +58,19 @@ type ImageInfo struct {
 	Labels      string
 }
 
+func getFreePort() string {
```

Comment:
Thanks for this cleanup.

Reply:
Wondering this needs to move into some internal/ or pkg/test/ so code doesn't get repeated.